core/src/main/scala/org/apache/spark/util/Utils.scala

```diff
@@ -2015,27 +2015,38 @@ private[spark] object Utils
     }
   }
 
+  private[util] val threadInfoOrdering = Ordering.fromLessThan {
```

Comment:
I prefer to use `PrivateMethodTester` during testing rather than expanding its scope.

Reply:
There is an existing method in `Utils`, seems not bad to keep same
```
private[util] def trimExceptCRLF(str: String): String
```